.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,9 @@
 <!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask in the Help Contributors room linked above. We're here to help! -->
```

Comment:
While you're at it, can you change "Help Contributors" to "Contributors"? 😄

Reply:
Will do! Thanks for pointing out.